### DISCUSSION TITLE
Any way to save/convert a Open3D pointcloud to a file format that Autodesk Recap can read?

### DISCUSSION THREAD

**Discussion by hradec** [isl-org/Open3D#6503] [General]
Did anyone ever tried this before? 

I'm having trouble with registration of pointcclouds created from RGBD photos taken with iOS devices (I'm saving the Depth as TIF 32 bit float for precision). 

Although I prefer to use open source software, I was asked to try to do the registration in Autodesk Recap, but for that I need to save the pointcloud XYZ and RGB data as a file format recap can read. 

These are the file formats Recap supports: [link](https://help.autodesk.com/view/RECAP/ENU/?guid=supported_file_formats)
```
Import		
==================================================
CL3 (Topcon)	LAS		RDS (3D only; Riegl)
CLR (Topcon)	PRJ (Leica)	TXT
E57		PTG (Leica)	XYB
FLS (Faro)	PTS		XYZ
FWS (Faro)	PTX		ZFS (Zoller+Fröhlich)
LSPROJ (Faro)	RCS		ZFPRJ (Zoller+Fröhlich)
```

**Comment by saurabheights** [comment]
Not an expert in IO formats, but you can try meshlab to convert ply to e57. Tested, works fine.

Open3D supports only following formats - http://www.open3d.org/docs/latest/tutorial/geometry/file_io.html#File-IO
